geode-docs/tools_modules/micrometer/micrometer-meters.html.md.erb

```diff
@@ -0,0 +1,97 @@
+
+Every meter in Micrometer has the following attributes, but may also have more than these depending on the meter type:
+
+-   `Name`: a lowercase, 'dot' delimited name
+-   `Description`: a short text description of the meter
+-   `Value`: the value of the meter, always a number
+-   `Units`: the unit of the value
+-   `Tags`: key/value string pairs to define facts about the meter
+
+### Meters supplied by Micrometer
+<%=vars.product_name%> has been instrumented with a set of Micrometer provided meters, called binders, which contain one or many individual meters with pre-defined types and tags.
+The following are a subset of the Micrometer binders that have been added to <%=vars.product_name%>:
+
+*JVM specific*
+
+-   JvmGcMetrics
+-   JvmMemoryMetrics
+-   JvmThreadMetrics
+
+*Operating System specific*
+
+-   ProcessorMetrics
+-   UptimeMetrics
+-   FileDescriptorMetrics
+
+### Meters specific to <%=vars.product_name%>
+There are <%=vars.product_name%> specific meters that have been instrumented to allow developers to monitor certain aspects of the <%=vars.product_name%> cluster. The namespace used for these metrics includes the Apache Geode (e.g. geode.*) product name as a means to distinguish this meter from others similar to it in downstream systems. The following meters have been added to <%=vars.product_name%>:
+
+| Meter:              | geode.cache.entries |
+|---------------------|---------------------|
+| Type:               | Gauge               |
+| Description:        | The current count of entries in the cache for this member (locators will not have this metric)|
+| Tag:                |      region: The name of the region associated with the count of entries|
+| Tag:                |      data.policy: The data policy described for these regions, see [RegionShortcuts for Peers and Servers](../../basic_config/data_regions/region_shortcuts.html#region_shortcuts__section_D0975C76572E41F79C1A6EE7CF371251) |
+
+| Meter:              | geode.function.executions |
+|---------------------|---------------------------|
+| Type:               | Timer                     |
+| Description:        | A total time and count for any execution that completed without errors of this function on this member. |
+| Tag:                | Function: The name or class of the function |
+| Tag:                |  Succeeded: true/false |
+
+| Meter:              | geode.gateway.receiver.events |
+|---------------------|-------------------------------|
+| Type:               | Counter                       |
+| Description:        | The total number of events received by this GatewayReceiver |
+
+| Meter:              | geode.cache.gets    |
+|---------------------|---------------------|
+| Type:               | Timer               |
+| Description:        | A total time and count for any get operation performed against the cache server for this member (locators will not have this metric) |
```

Comment:
Actually, `geode.cache.gets` currently only records gets from a Java or native client. The description as written is behavior we'd like to have, but we're not quite there yet with the implementation.

Reply:
clarified.